app/src/main/res/values/styles.xml

```diff
@@ -39,4 +39,7 @@
         <item name="android:padding">@dimen/layout_padding_8dp</item>
     </style>
 
+    <style name="DialogTheme" parent="Theme.AppCompat.Light.Dialog.Alert">
```

Comment:
@mayank8318 Why don't you use this style https://github.com/mayank8318/self-service-app/blob/bd0c29186e5af18720ccfc8dcf7c246760c12f76/app/src/main/res/values/styles.xml#L28

Reply:
@therajanmaurya that is using colorAccent = @color/primary but the dialog should use the accent color. I didn't change that because that style might be used somewhere else so I created a new style.
Should I make changes in that style itself?

Reply:
@mayank8318 that style is using in Material Dialog builder class in /utils. So do one thing, rename the style with `MaterialDatePickerTheme` and we will enhance this style if needed.